src/A2A/JsonRpc/ErrorTypes.cs

```diff
@@ -13,7 +13,7 @@ public class A2AClientException : Exception
     /// <param name="message">The error message.</param>
     /// <param name="innerException">Optional inner exception.</param>
     public A2AClientException(string message, Exception? innerException = null)
-        : base(message, innerException)
+        : base(string.IsNullOrEmpty(message) ? throw new ArgumentNullException(nameof(message)) : message, innerException)
```

Comment:
The base `Exception` type allows for `null` message:

https://github.com/dotnet/runtime/blob/1fb517811e25a96a92ce377d51bf693461ae4f75/src/libraries/System.Private.CoreLib/src/System/Exception.cs#L36

Please let me know if you believe we should also allow for nulls here, in such case I am going to make the argument nullable